Interpreting a casted term in the scope bound to its type if any (#6134)
coq/coq: doc/sphinx/user-extensions/syntax-extensions.rst

   then :g:`a` of type :g:`t` in :g:`f t a` is not recognized as an argument to
   be interpreted in scope ``scope``.

>    In explicit casts :g:`term : coercion_class`, the :g:`term`
>    will also be interpreted in :token:`scope_name`.

Reviewer: ```suggestion
   In explicit casts :n:`@term : @coercion_class`, the :n:`@term`
   is interpreted in the :token:`scope_name` associated with :n:`@coercion_class`.
```

:n: shows syntax with hyperlinks and in green
Seemed like "also" didn't add anything.

Reviewer 2: Thanks, I'm apparently not editing the doc often enough and forgot those things every time.

> Seemed like "also" didn't add anything.

Indeed, it's probably more confusing than useful.

> 

Reviewer: If this applies to all casts (not just coercion_classes), then this should also be mentioned where `:>` is defined, i.e. https://coq.inria.fr/refman/language/core/definitions.html#type-cast

Reviewer 2: Good point, done.
